src/pages/billing/Project.js

```diff
@@ -241,6 +243,197 @@ const OtherMessaging = ({ cost }) => {
   ])
 }
 
+const GcpBillingAccountControls = ({
```

Comment:
I moved this component out of the very large ProjectDetail component. It will not be shown at all for Azure-backed projects, and keeping all of its methods and state together helps make the code easier to understand.

Reply:
Thank you!